electron/js/menu/system.js

```diff
@@ -28,6 +30,11 @@ const locale = require('./../../locale/locale');
 let menu;
 var menuTemplate;
 
+const launcher = new autoLaunch({
+  name: 'Wire',
```

Comment:
The tray menu uses config.Name. Should I use that or the pkg.Name as per your comment?